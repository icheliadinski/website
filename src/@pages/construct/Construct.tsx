import React from 'react'

export const Construct = () => {
  return <div>Under Construct</div>
}
