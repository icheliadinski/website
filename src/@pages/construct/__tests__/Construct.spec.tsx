import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import { Construct } from '../Construct'

describe('[Construct Page]', () => {
  let wrapper: ShallowWrapper

  beforeEach(() => {
    wrapper = shallow(<Construct />)
  })

  it('renders successfully', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
