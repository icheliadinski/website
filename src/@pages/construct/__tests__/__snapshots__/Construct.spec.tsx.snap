// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Construct Page] renders successfully 1`] = `
<div>
  Under Construct
</div>
`;
